### ISSUE TITLE
CGB is using deprecated plugin

### ISSUE THREAD

**Issue by espenjanson** [ahmadawais/create-guten-block#132]
CGB should use mini css extract plugin instead of extract... since extract... is deprecated.

That's it, that's all folks!

Thanks for an awesome project :)

**Comment by ahmadawais** [comment]
Yes, a lot can be improved here with create-guten-block but right now it just works. Nothing's broken. For updating create-guten-block to all these latest tooling I need more bandwidth and all the help I can get. Feel free to contribute, happy to accept PR's.
